### PR TITLE
First pass at editing NEWS for release; make myself maintainer

### PR DESCRIPTION
**Pull request by jennybc**
I'm merging this @hadley but feel free to comment on NEWS.md. I've started to re-organize for release and can make changes.

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,74 +1,74 @@
...
+    * Eliminates a subtle source of disagreement between user-provided column names and guessed column types (#169, #81).
+    * Leading or embedded empty columns are no longer automatically dropped, regardless of whether there is a column name. (#157, #261)
+    * Worksheets that are completely empty or that contain only column names no longer error, but return a tibble with zero rows. (#222, #144, #65)
+    * Improved handling of leading and embedded blank rows and explicit row skipping. (#224, #194, #178, #156, #101)
 
-* Selective column type guessing: `col_types` now accepts `"guess"` to allow user to specify some column types, while allowing others to be guessed (#286 @jennybc)
+* User-supplied `col_names` are processed relative to user-supplied `col_types`, if given. Specifically, `col_names` is considered valid if it has the same length as `col_types`, before *or after* removing skipped columns. (#81, #261)
 
-* Numeric data that appears in a `"date"` column is coerced to a date. Also throws a warning. (#277, #266 @jennybc)
+## Column types and coercion
```
Comment (hadley) [inline NEWS.md]: This section would be slightly easier to scan if alphabetised

File: DESCRIPTION [hunk 2]
```diff
@@ -2,8 +2,8 @@ Package: readxl
 Title: Read Excel Files
 Version: 0.1.1.9000
 Authors@R: c(
-    person("Hadley", "Wickham", ,"hadley@rstudio.com", role = c("aut", "cre")),
-    person("Jennifer", "Bryan", ,"jenny.f.bryan@gmail.com", role = c("aut")),
+    person("Hadley", "Wickham", ,"hadley@rstudio.com", role = c("aut")),
+    person("Jennifer", "Bryan", ,"jenny.f.bryan@gmail.com", role = c("aut", "cre")),
```
Comment (hadley) [inline DESCRIPTION]: Use your RStudio address?

File: NEWS.md (hunk 1 above)
Comment (jennybc) [inline NEWS.md]: There is some logic to the ordering, apparently quite subtle. I started with new col types, then 'other news about col types', then coercion/extraction stuff. I have bulleting problems which will get fixed and may also help.

File: DESCRIPTION (hunk 2 above)
Comment (jennybc) [inline DESCRIPTION]: changed

**Pull request merged**